### ISSUE TITLE
Undefined variable: self at * Upgrade.php#245

### ISSUE THREAD

**Issue by ho4ho** [owncloud/core#35773]
### Steps to reproduce
1. Have installed 10.2.0
2. Use 10.2.1 files
3. Use ./occ upgrade
4. owncloud.log check

### Expected behaviour
No entry ``Undefined`` log

### Actual behaviour
Entry ``Undefined`` log

### Server configuration
**Operating system**: Debian

**Web server:** Nginx

**Database:** MySQL 

**PHP version:** 7.2.19

**ownCloud version:** 10.2.1

**Updated from an older ownCloud or fresh install:** Updated

**Where did you install ownCloud from:** tar.bz2

**Signing status (ownCloud 9.0 and above):** All o.k.

**The content of config/config.php:**

Non-relevant

**List of activated apps:**

Non-relevant

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
Non relevant

#### Web server error log
Non relevant

#### ownCloud log (data/owncloud.log)
```
{"reqId":"7MK26PEod2c4Va6bfJwy","level":3,"time":"2019-07-04T19:50:50+02:00","remoteAddr":"","user":"--","app":"PHP","method":"--","url":"--","message":"Undefined variable: self at \/var\/www\/owncloud\/core\/Command\/Upgrade.php#245"}
{"reqId":"7MK26PEod2c4Va6bfJwy","level":0,"time":"2019-07-04T19:50:50+02:00","remoteAddr":"","user":"--","app":"core","method":"--","url":"--","message":"starting upgrade from 10.2.0.5 to 10.2.1.4"}
{"reqId":"7MK26PEod2c4Va6bfJwy","level":3,"time":"2019-07-04T19:50:51+02:00","remoteAddr":"","user":"--","app":"PHP","method":"--","url":"--","message":"str_repeat(): Second argument has to be greater than or equal to 0 at \/var\/www\/owncloud\/lib\/composer\/symfony\/console\/Helper\/ProgressBar.php#509"}
```

#### Browser log
Non relevant

**Comment by phil-davis** [comment]
This code was fixed in `master` https://github.com/owncloud/core/pull/35774/files#diff-327e70af620b69472bda67f9e2885742 but it has not yet been backported - it was part of a long list of things found by `phpstan`

Backport of `phpstan` changes is proposed in PR #35774 
If that takes too long to get passing and reviewed/tested etc, then I will make a small PR just for this particular fix.

**Comment by phil-davis** [comment]
Fixed in PR #35774 and should end up in the next release.
